source/_docs/blueprint/schema.markdown

```diff
@@ -72,9 +72,10 @@ homeassistant:
   keys:
     min_version:
       description: >
-        Minimum required version of Home Assistant to use the blueprint. For example,
-        `2022.4.0`. It is important to set this if the blueprint uses any features
-        introduced in recent releases to head off issues.
+        Minimum required version of Home Assistant to use the blueprint in the format of
+        *major*.*minor*.*patch* (all parts are required). For example, `2022.4.0`. It is
```

Comment:
Hmm. This seems incorrect and more likely a bug. Did you report it before making this change?


../Frenck

Reply:
I didn't open a bug first. From the error message I got back in the automations associated to the blueprint (below) I assumed it was desired behaviour.

> There is an error in this Blueprint: Invalid blueprint: Version needs to be formatted as {major}.{minor}.{patch} for dictionary value @ data['blueprint']['homeassistant']['min_version']. Got '2024.6'